src/app/core/services/ball-dont-lie.service.ts

```diff
@@ -1,26 +1,34 @@
 import { HttpClient, HttpParamsOptions } from '@angular/common/http';
 import { Injectable } from '@angular/core';
-import { Observable, switchMap, catchError, throwError, filter, map, of } from 'rxjs';
+import { Observable, switchMap, catchError, throwError, filter, map, of, Subject, tap } from 'rxjs';
+import { ITeam } from '../models/ITeam';
 
 @Injectable({
   providedIn: 'root'
 })
 export class BallDontLieService {
-  ballDontLieUrl: string = 'https://balldontlie.io';
+  ballDontLieUrl: string = 'https://www.balldontlie.io';
 
   constructor(
     private httpService: HttpClient
   ) { }
 
   // TODO: Make a Model for an NBA team to type this properly:
-  getAllNBATeams(): Observable<any> {
-    return this.httpService.get(`${this.ballDontLieUrl}/api/v1/teams`).pipe(
-      catchError((err: any) => {
-        console.log(err);
-        return throwError('Error in BallDontLieService GET All Teams method');
-    })
-    )
+  getAllNBATeams(): Observable<ITeam[]> {
+    return this.httpService.get<any>(`${this.ballDontLieUrl}/api/v1/teams`)
+    .pipe(
+      map(function(response){
+        return <ITeam[]>response.data;
+      }),
+      tap(t => console.log(t)),
```

Comment:
Don't forget to remove console.log's.  Totally fine to keep it for our play project though!